src/debug_toolkit/main.py

```diff
@@ -207,10 +207,14 @@ def memory(pid: int, seconds: int = 60, verbose: bool = False):
     inject_string(pid, payload, trampoline=True, trampoline_timeout=timeout_seconds, verbose=verbose)
 
 @app.command()
-def stack_trace(pid: int, all_threads: bool = True, verbose: bool = False):
+def stack_trace(pid: int, all_threads: bool = True, amount: int = 1, sleep_duration_s: int=0, verbose: bool = False):
+    if amount < 1 or sleep_duration_s < 0:
+        typer.secho(f"ERROR! amount must be greater than 1 and sleep must be greater than 0", fg="red")
```

Comment:
I think you want to return here
I think the min sleep time should be 1, and I think that should also be the default